### PR TITLE
Team support for FAH Client

### PR DESCRIPTION
**Pull request by wjimenez5271**

### CODE REVIEW

File: charts/folding-at-home/v0.1.0/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    <config>
+      <!--
+        To set your user name, team and passkey just edit the text
+        in quotes below.
+      -->
+
+      <!-- User Information -->
+      <user value=""/> <!-- Enter your user name here -->
+      <team value=""/>         <!-- Your team number -->
+      <passkey value=""/>       <!-- 32 hexadecimal characters if provided -->
```
Comment (davidnuzik) [inline charts/folding-at-home/v0.1.0/templates/deployment.yaml:126]: Nit: passkey is optional. Think perhaps should tweak the comment to make this more clear?
Do we need to explain what the passkey is/does? People shouldn't need it unless when they created their (also optional) user they opted into making one.
IIRC the passkey is essentially something you can opt into creating for your user so others can't claim your work (not sure how that could happen though).

Comment (wjimenez5271) [reply charts/folding-at-home/v0.1.0/templates/deployment.yaml:126]: good point, i'll add that.

File: charts/folding-at-home/v0.2.0/questions.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    default: ""
+    description: "Folding at Home Teamname"
+    type: string
+    label: FAH Teamname (optional)
+    group: "Folding @ Home Team Info"
+  - variable: fah.passkey
+    default: ""
+    description: "Folding at Home Passkey"
+    type: string
+    label: FAH Passkey if you set one. FAH accounts do not require passkeys, this is optional
```
[the remark this answers is not in the document]
Comment (davidnuzik) [reply charts/folding-at-home/v0.2.0/questions.yaml:58]: Fine by me. Thanks!

**Comment by davidnuzik** [comment]
> Also FYI you were commenting in the 0.1 version of the chart, 0.2 has the variables supported

Ah my bad. Curious, why do I see so many files changed?

**Comment by wjimenez5271** [comment]
@deniseschannon ready for your review

**Pull request merged**